packages/bvaughn-architecture-demo/components/sources/CurrentLineHighlight.module.css

```diff
@@ -17,7 +17,6 @@
 }
 
 .CurrentSearchResult {
-  background: var(--background-color-current-search-result);
```

Comment:
No need to highlight the full line anymore. That was a stop-gap until we were highlighting tokens within the line.